.rubocop_todo.yml

```diff
@@ -118,7 +118,7 @@ RSpec/FilePath:
 # Offense count: 56
 # This cop supports safe autocorrection (--autocorrect).
 RSpec/HooksBeforeExamples:
-  Enabled: false
+  Enabled: true
```

Comment:
Let's just remove these, they are enabled by default and this information is now stale anyway

Reply:
done.